bin/bundle

```diff
@@ -1,5 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
this is why bin/bundle didn't work... the special #! comment has to go very first.